src/secondary.jl

```diff
@@ -0,0 +1,197 @@
+export rundssp!, rundssp, runstride!, runstride, sscode!, sscode, helixselector, sheetselector, coilselector, sscodeselector
+
+const dssp_executable = `$(DSSP_jll.mkdssp()) --mmcif-dictionary $(DSSP_jll.mmcif_pdbx_dic) --output-format dssp`
+const stride_executable = `$(STRIDE_jll.stride_exe())`
+const dssp_init_line = "  #  RESIDUE AA STRUCTURE BP1 BP2  ACC     N-H-->O    O-->H-N    N-H-->O    O-->H-N    TCO  KAPPA ALPHA  PHI   PSI    X-CA   Y-CA   Z-CA"
+const header_line = "HEADER"
+const helixsscodes = Set(["G", "H", "I", "P"])
+const sheetsscodes = Set(["E", "B"])
+const coilsscodes = Set(["C", "T", "S", " "])
+
+"""
+    rundssp(curr_model::Model, mod_n::Int)
+"""
+function rundssp(curr_model::Model, mod_n::Int)
+    struc = ProteinStructure()
```

Comment:
It may be sufficient to `writepdb(io, curr_model)` without the `struc` setup.

Reply:
Yes, I just tested this.